docs/orchestrating-elastic-stack-applications/stack-config-policy.asciidoc

```diff
@@ -242,3 +316,30 @@ In order to avoid a conflict between multiple Elasticsearch clusters writing the
 - sets the `base_path` to `snapshots/<namespace>-<esName>` when it is not provided, for Azure, GCS and S3 repositories
 - appends `<namespace>-<esName>` to `location` for a FS repository
 - appends `<namespace>-<esName>` to `path` for an HDFS repository
+
+[float]
+[id="{p}-{page_id}-specifics-secret-mounts"]
+== Specifics for secret mounts
+
+In ECK `2.11.0` a new field was introduced to the Elastic Stack configuration policy under the `elasticsearch` field called `secretMounts`.
+This field allows users to specify a user created secret and a mountPath to indicate where this secret should be mounted in the Elasticsearch Pods that are managed by the Elastic Stack configuration policy.
+This field can be used to add additional secrets to the Elasticsearch Pods that may be needed for sensitive files required for configuring Elasticsearch realms.
+The secret should be created by the user is the same namespace as the operator.
+The operator will read this secret and copy it over to the namespace of Elasticsearch so that it can be mounted by the Elasticsearch Pods.
```

Comment:
We used to avoid future tense, but I'm not a very skilled doc writer.

Reply:
makes sense, will change this.

Reply:
I changed the sentence to not be future tense.